### PR TITLE
dev-es sync with main

### PR DESCRIPTION
**Pull request by krol3**
### Describe your changes

Sync dev-es branch with main

cc: @electrocucaracha , @ramrodo , @92nqb , @seokho-son

### CODE REVIEW

**Review by electrocucaracha — APPROVED** [review]
/lgtm

**Review by ramrodo — APPROVED** [review]
LGTM

File: content/es/_index.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Para una lista de contribuidores completa, consulte [esta página de GitHub](htt
 
 Esta versión del glosario Cloud Native en español incluye contribuciones realizadas por
 [Victor Morales](https://www.linkedin.com/in/electrocucaracha/),
-[Rael Garcia Arnes](https://www.linkedin.com/in/rael/),
+[Mariano Gabriel González](https://www.linkedin.com/in/marianogg9/),
```
Comment (seokho-son) [inline content/es/_index.md:38]: minor question to the team.

I hope to check the name needs to be replaced (agreed in the team?)
I think this section is to acknowledge major contributors so does not need to be aligned with the reviewers/approvers. (the team can add many more if the team think they are major contributors)

**Comment by seokho-son** [comment]
By the way, it is nice work! the other parts are LGTM!
Thanks to the team!

File: content/es/_index.md (hunk 1 above)
Comment (krol3) [reply content/es/_index.md:38]: Thanks @seokho-son for the comments, I will double check with the team, I think it was an agreement with the team too. cc: @electrocucaracha @ramrodo

Comment (ramrodo) [reply content/es/_index.md:38]: Yes. We agree with the change. We're not aligned with the reviewers/approvers, just adding a major contributor and replacing an inactive contributor for now.

Comment (electrocucaracha) [reply content/es/_index.md:38]: Well, @raelga was crucial for the team, but he has taken some personal time lately. In the other hand, @marianogg9 has been doing an amazing work during this year. BTW, this is a life document and we can adjust it later.

**Comment by jihoon-seo** [comment]
@krol3 Thank you for your work!
Of course you can create a feature branch from `dev-es` and merge this feature branch to `main`, but as you already know, we can first check this link: https://github.com/cncf/glossary/compare/main...dev-es, and if GitHub UI says that there is no merge conflict ("Able to merge.") when merging `dev-es` to `main`, then we can just open a PR that merges `dev-es` to `main` directly!
With this way we don't need to create a merge commit (like 894715d in this PR), and this is the way that Seokho and I recommend. 😊

**Comment by krol3** [comment]
> @krol3 Thank you for your work! Of course you can create a feature branch from `dev-es` and merge this feature branch to `main`, but as you already know, we can first check this link: [main...dev-es](https://github.com/cncf/glossary/compare/main...dev-es), and if GitHub UI says that there is no merge conflict ("Able to merge.") when merging `dev-es` to `main`, then we can just open a PR that merges `dev-es` to `main` directly! With this way we don't need to create a merge commit (like [894715d](https://github.com/cncf/glossary/commit/894715d3e1edf114aec2969fec886d9b14055804) in this PR), and this is the way that Seokho and I recommend. 😊

Nice tip! I will annotate here for the next merge, or put in the documentation. Thank you @seokho-son

**Review by seokho-son — APPROVED** [review]
LGTM!!

Let me merge this PR.

**Comment by seokho-son** [comment]
By the way, thanks to the Spanish team for managing even the commit history! It looks nice.

**Pull request merged**